electron/index.js

```diff
@@ -316,8 +326,10 @@ async function writeFileSafe(filename, contents) {
 }
 
 ipcMain.handle("fs-job", async (event, job) => {
-    const filenameSafe = job.filename.replace(/[^a-z\.\-_0-9]/gi, "_");
-    const fname = path.join(storePath, filenameSafe);
+    const filenameSafe = path.isAbsolute(job.filename)
+        ? job.filename
+        : job.filename.replace(/[^a-z\.\-_0-9]/gi, "_");
+    const fname = path.isAbsolute(filenameSafe) ? filenameSafe : path.join(storePath, filenameSafe);
```

Comment:
Doesn't this introduce a security vulnerability? See https://github.com/tobspr-games/shapez.io/pull/1375